utils/sources.ts

```diff
@@ -356,6 +356,112 @@ const additional: Source = {
   ],
 };
 
+const info: Source = {
+  type: 'Information',
+  entries: [
+    {
+      name: 'What is DNS?',
+      url: 'https://www.techtarget.com/searchnetworking/definition/domain-name-system',
+      by: 'Tech Target',
+    },
+    {
+      name: 'How DNS works?',
+      url: 'https://www.cloudflare.com/en-gb/learning/dns/what-is-dns/',
+      by: 'CloudFlare',
+    },
+    {
+      name: 'DNS',
+      url: 'https://www.javatpoint.com/computer-network-dns',
+      by: 'JavaTPoint',
+    },
+    {
+      name: 'DNS in Application Layer',
+      url: 'https://www.geeksforgeeks.org/domain-name-system-dns-in-application-layer/',
+      by: 'Geeks For Geeks',
+    },
+    {
+      name: 'More About DNS',
+      url: 'https://www.infoblox.com/glossary/domain-name-system-dns/',
+      by: 'Infoblox',
+    },
+    {
+      name: 'What is Internet Protocols?(IP)',
+      url: 'https://www.cloudflare.com/en-gb/learning/network-layer/internet-protocol/',
+      by: 'CloudFlare',
+    },
+    {
+      name: 'Types of Internet Protocols',
+      url: 'https://www.geeksforgeeks.org/types-of-internet-protocols/',
+      by: 'Geeks For Geeks',
+    },
+    {
+      name: 'Internet Protocols',
+      url: 'https://www.techtarget.com/searchunifiedcommunications/definition/Internet-Protocol',
+      by: 'Tech Targets',
+    },
+    {
+      name: 'What is IP?',
+      url: 'https://www.javatpoint.com/ip',
+      by: 'JavaTPoint',
+    },
+    {
+      name: 'Types of IP',
+      url: 'https://www.usg.edu/galileo/skills/unit07/internet07_03.phtml',
+      by: 'Library Learning',
+    },
+    {
+      name: 'How does Web Browsers work?',
+      url: 'https://medium.com/@monica1109/how-does-web-browsers-work-c95ad628a509#:~:text=High%2Dlevel%20architecture%20of%20browser&text=The%20Browser%20Engine%3A%20The%20browser,and%20manipulates%20the%20rendering%20engine.',
+      by: 'Medium',
+    },
+    {
+      name: 'How browsers work?',
+      url: 'https://web.dev/howbrowserswork/',
```

Comment:
amazing article.
Change Title to: **Internal Operations of Browser Engines** and move ot position 5.